official/projects/fffner/fffner_prediction.py

```diff
@@ -0,0 +1,328 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""FFFNER prediction task."""
+import dataclasses
+from collections import defaultdict
+
+from absl import logging
+import numpy as np
+import tensorflow as tf
+
+from official.core import base_task
+from official.core import config_definitions as cfg
+from official.core import task_factory
+from official.modeling import tf_utils
+from official.modeling.hyperparams import base_config
+from official.nlp.configs import encoders
+from official.nlp.data import data_loader_factory
+from official.nlp.tasks import utils
+
+from official.projects.fffner import fffner_classifier
+
+METRIC_TYPES = frozenset(
+    ['accuracy', 'matthews_corrcoef', 'pearson_spearman_corr'])
+
+
+@dataclasses.dataclass
+class FFFNerModelConfig(base_config.Config):
+  """A classifier/regressor configuration."""
+  num_classes_1: int = 0
```

Comment:
What are num_classes_1 and num_classes_2? 

(1) Google codebase don't recommend 1 and 2 as variable names
(2) better to have a more meaningful name to describe what it is directly